docs/index.md

```diff
@@ -0,0 +1,17 @@
+# Welcome to MkDocs
+
+For full documentation visit [mkdocs.org](https://www.mkdocs.org).
+
+## Commands
+
+* `mkdocs new [dir-name]` - Create a new project.
```

Comment:
Please make this an example of what a library page would look like.

Reply:
DONE